Parsers/InfobloxNIOS/InfobloxNIOS.txt

```diff
@@ -27,21 +27,14 @@
 //
 let RawData = Syslog 
     | where Computer in ("datasource") and Facility == "local6"
-    | extend Parser = extract_all(@"^([a-zA-Z]{3}\s\d{2}\s[0-9\:]+)\s(\S+)\s([a-z_]+)(\[(\d+)\]|\:)(\:\s)?(.*)", dynamic([1,2,3,4,5,6,7]), SyslogMessage)
+    | extend Parser = extract_all(@"^(\d{2}\-[a-zA-Z]{3}\-\d{4}\s[0-9\.\:]+)?\s?([a-zA-Z-_]+)(\s|\:)?(.*)", dynamic([1,2,3,4]), SyslogMessage)
     | mv-expand todynamic(Parser)
-    | extend logTime = tostring(Parser[0]), 
-        ServerIP = tostring(Parser[1]), 
-        LogType = tostring(Parser[2]), 
-        PID = toint(Parser[4]),
-        subString = tostring(Parser[6])
-    | extend Parser2 = extract_all(@"^(\d{2}\-[a-zA-Z]{3}\-\d{4}\s[0-9\.\:]+)?\s?([a-zA-Z-_]+)(\s|\:)?(.*)", dynamic([1,2,3,4]), subString)
-    | mv-expand todynamic(Parser2)
-    | extend responseTime = todatetime(Parser2[0]),
-            Type = tostring(Parser2[1]),
-            RawData_subString = tostring(Parser2[3])
-    | project-away Parser,Parser2,subString;
+    | extend responseTime = todatetime(Parser[0]),
+            Type = tostring(Parser[1]),
```

Comment:
Original Syslog also has Type field in schema which is overlapping with new field Type  from parser

Reply:
I have changed this field in the parser to Log_Type